README.md

```diff
@@ -1,140 +1,126 @@
+For Debian/Ubuntu, use the following commands:
 
-```shell script
-detect-secrets audit .secrets.baseline
+```shell
+sudo apt-get install cookiecutter
 ```
 
-When you run this command, you'll enter an interactive console and be presented with a list of high-entropy string /
-anything which _could_ be a secret, and asked to verify whether or not this is the case. By doing this, the hook will
-be in a position to know if you're later committing any _new_ secrets to the repo and it will be able to alert you
-accordingly.
+Otherwise, you can install `cookiecutter` via `pip` — you may wish to create a virtual environment first:
+
+```shell
+python3 -m pip install --user cookiecutter
+```
 
-### If `pre-commit` detects secrets during commit:
+## Changes to make post-creation
 
-If `pre-commit` detects any secrets when you try to create a commit, it will detail what it found and where to go to
-check the secret.
+Here's a few changes you **must** do once you've created your new project:
 
-If the detected secret is a false-positive, you should update the secrets baseline through the following steps:
+- Set up a Python virtual environment — there are **many** ways to set up a virtual environment, so we'll let you
+  decide what's best for you!
+- Git is not set up by default — open your terminal, navigate to your new project, run `git init` to set it up
 
-- Run `detect-secrets scan --update .secrets.baseline` to index the false-positive(s);
-- Next, audit all indexed secrets via `detect-secrets audit .secrets.baseline` (the same as during initial set-up, if a
-secrets baseline doesn't exist); and
-- Finally, ensure that you commit the updated secrets baseline in the same commit as the false-positive(s) it has been
-updated for.
+## Changes to consider post-creation
 
-If the detected secret is actually a secret (or other sensitive information), remove the secret and re-commit. There is
-no need to update the secrets baseline in this case.
+Here's a few changes you should consider changing once you've created your new project:
 
-If your commit contains a mixture of false-positives and actual secrets, remove the actual secrets first before
-updating and auditing the secrets baseline.
+- Make sure the `README.md` reflects what **you** want to do with your project!
+- Have a look inside the `docs/aqa` folder; you may want to modify some of them, e.g. the AQA plan
+- Want to add some project-specific checklists to the pull/merge request template? See the relevant Markdown files
+  within the `.github` (GitHub) or `.gitlab/merge_request_templates` (GitLab) folder
 
-### Note on Jupyter notebook cleaning
+## Licence
 
-It may be necessary or useful to keep certain output cells of a Jupyter notebook, for example charts or graphs
-visualising some set of data. To do this, add the following comment at the top of the input block:
+Unless stated otherwise, the codebase is released under the MIT License. This covers both the codebase and any sample
```

Comment:
I've seen projects based-off other projects stating they inherit the license from the parent project. I checked Driven Data's repo and that's also MIT. Do we need to do likewise and state this inherits the MIT license from the parent project?

Reply:
I think MIT is permissive, and let's you adopt a different licence to any parent project (see [here](https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licences#General_comparison)). In any case, I've acknowledged things directly used from Driven Data's cookiecutter later on in the `README.md`.